pkg/tagger/README.md

```diff
@@ -16,18 +16,20 @@ in their process. Switch between local and client mode will be done via a build
 ### Collector
 A **Collector** connects to a single information source and pushes **TagInfo**
 structs to a channel, towards the **Tagger**. It can either run in streaming
-mode or pull mode, depending of what's most efficient for the data source:
+mode, pull or fetchonly mode, depending of what's most efficient for the data source:
 
 #### Streamer
 The **DockerCollector** runs in stream mode as it collects events from the docker
 daemon and reacts to them, sending updates incrementally.
 
 #### Puller
-The **KubernetesCollector** and **ECSCollector** will run in pull mode as they
-need to query and filter a full entity list every time. They will only push
+The **KubernetesCollector** will run in pull mode as it needs to query and filter a full entity list every time. It will only push
 updates to the store though, by keeping an internal state of the latest
 revision.
 
+#### FetchOnly
+The **ECSCollector** will not push the updates to the Store. It is not meant to run in standalone. DockerCollector will trigger deletions in the store.
```

Comment:
The **ECSCollector** does not push updates to the Store by itself, but is only triggered on cache misses. As tasks don't change after creation, there's no need for periodic pulling. It is designed to run alongside DockerCollector, that will trigger deletions in the store.